src/main/java/edu/harvard/iq/dataverse/DatasetFieldServiceBean.java

```diff
@@ -667,9 +668,14 @@ private JsonObject filterResponse(JsonObject cvocEntry, JsonObject readObject, S
                             param = param.substring(1);
                             String[] pathParts = param.split("/");
                             logger.fine("PP: " + String.join(", ", pathParts));
-                            JsonValue curPath = readObject;
-                            vals.add(i, processPathSegment(0, pathParts, curPath, termUri));
-                            logger.fine("Added param value: " + i + ": " + vals.get(i));
+                            var foundPart = processPathSegment(0, pathParts, readObject, termUri);
+                            if (foundPart == null) {
+                                nrOfNotFound ++ ;
+                                logger.warning("External Vocabulary: no value found for %s - %s".formatted(filterKey, param));
```

Comment:
Thinking about adding this info to a list or map for the last warning and reduce the other warnings to fine logging. That clogs the log less.

Reply:
On second thought not worth the trouble